internal/pkg/agent/application/coordinator/coordinator.go

```diff
@@ -412,6 +418,22 @@ func (c *Coordinator) State() State {
 	return c.stateBroadcaster.Get()
 }
 
+// CoordinatorActive is a blocking method that waits for a channel response
+// from the coordinator loop. This can be used to as a basic health check,
+// as we'll timeout and return false if the coordinator run loop doesn't
+// respond to our channel.
+func (c *Coordinator) CoordinatorActive(timeout time.Duration) bool {
```

Comment:
Why is this prefixed with `Coordinator`? It is the coordinator, why not just `Active()`?

Reply:
yeah, good point